testsuite/run.py

```diff
@@ -13,32 +13,31 @@
 
 import e3.testsuite
 import e3.testsuite.driver
+from e3.testsuite.result import TestStatus
 
 
 from drivers.python_script import PythonScriptDriver
 
 
 class Testsuite(e3.testsuite.Testsuite):
-    TEST_SUBDIR = 'tests'
-    DRIVERS = {'python-script': PythonScriptDriver}
+    tests_subdir = 'tests'
+    test_driver_map = {'python-script': PythonScriptDriver}
 
-    @property
-    def default_driver(self):
-        return 'decoder'
+    def set_up(self):
+        super().set_up()
+
+        # Some tests rely on an initially empty GPR_PROJECT_PATH variable
```

Comment:
Is this something new? Which test relies on that?

Reply:
The `alr setenv` ones will have unexpected output when run with `GPR_PROJECT_PATH`, I had that once.

Reply:
Probably new, as I found that only today. That was for the `setenv/with-external` testcase